build.gradle

```diff
@@ -8,8 +8,8 @@ plugins {
     id 'org.hidetake.ssh' version "2.9.0"
 }
 
-sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
I'll try to find a way to avoid this

Reply:
That was going to be my first question!

Reply:
The final code didn't need it at all - one of the stream toList() methods does require Java 16 at least.

Reply:
Dangit, intellij lied to me. More changes needed :(